rasa/core/train.py

```diff
@@ -118,7 +118,7 @@
                         file_importer,
                         train_path,
                         policy_config=policy_config,
-                        exclusion_percentage=current_run,
+                        exclusion_percentage=percentage,
```

Comment:
I missed this, when merging the original PR. It is part of another PR, that is not yet merged. Just wanted to fix it with the release.